### PR TITLE
[doc] add the Navitia POI specification

### PR DESCRIPTION
**Pull request by woshilapin**
I mostly made a copy/paste from a private documentation I had. Only adapted some of it to Markdown (I might have missed some).

### CODE REVIEW

File: poi_fr.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+Les catégories dans Navitia sont standardisées en utilisant OSM comme source de
+données. Les POI pouvant avoir plusieurs "tags" utilisés comme source de
+catégories Navitia, ces tags sont priorisées de la manière suivante :
+
+- utilisation du tag `amenity` (lien vers la [description OSM](http://wiki.openstreetmap.org/wiki/Key:amenity))
+- utilisation du tag `tourism` (lien vers la [description OSM](http://wiki.openstreetmap.org/wiki/Key:tourism))
+- utilisation du tag `leisure` (lien vers la [description OSM](http://wiki.openstreetmap.org/wiki/Key:leisure))
+- utilisation du tag `aeroway` (lien vers la [description OSM](http://wiki.openstreetmap.org/wiki/Key:aeroway))
+- utilisation du tag `railway` (lien vers la [description OSM](http://wiki.openstreetmap.org/wiki/Key:railway))
+- utilisation du tag `shop` (lien vers la [description OSM](http://wiki.openstreetmap.org/wiki/Key:shop))
```
Comment (woshilapin) [inline poi_fr.md]: I've removed a part here about a big SQL query that didn't make so much sense... Please yell at me if you think it needs to stay here. If yes, please help me with some comments about what is this query and why it is here.

File: poi_fr.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+    {"amenity:post_office" , new ed::types::PoiType(4, "bureau de poste")},
+    {"amenity:bicycle_rental" , new ed::types::PoiType(5, "station vls")},
+    {"amenity:bicycle_parking" , new ed::types::PoiType(6, "Parking vélo")},
+    {"amenity:parking" , new ed::types::PoiType(7, "Parking")},
+    {"amenity:police" , new ed::types::PoiType(8, "Police, Gendarmerie")},
+    {"amenity:townhall" , new ed::types::PoiType(9, "Mairie")},
+    {"leisure:garden" , new ed::types::PoiType(10, "Jardin")},
+    {"leisure:park" , new ed::types::PoiType(11, "Zone Parc. Zone verte ouverte, pour déambuler. habituellement municipale")}
+};
+```
```
Comment (woshilapin) [inline poi_fr.md]: I've removed a last sentence here that makes a link with another internal documentation. It's possible that we could replace this link with something that points to [that](https://doc.navitia.io/#street-network-objects) but I'm not sure.

File: poi_fr.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+La structure des données est la même que celle utilisée pour le format NTFS
+pour les horaires.  La seule différence est le séparateur ";".
+
+# Liste des fichiers
+Fichier | Contrainte | Commentaire
+--- | --- | ---
+[`poi_type.txt`](#poi_typetxt-requis) | Requis | Ce fichier liste des types de lieux associés aux POI de l'export
+[`poi.txt`](#poitxt-requis) | Requis | Ce fichier contient tous ou partie des POI à ajouter au référentiel
+[`poi_properties.txt`](#poi_propertiestxt-optionnel) | Optionnel | Ce fichier contient les propriétés complémentaires des POI de l'export. Attention, même si ce fichier est optionnel, certaines propriétés ont des impacts directs sur les itinéraires proposés ou le bon fonctionnement de certaines fonctionnalités du NMP.
```
Comment (patochectp) [inline poi_fr.md]:
```suggestion
[`poi_properties.txt`](#poi_propertiestxt-optionnel) | Optionnel | Ce fichier contient les propriétés complémentaires des POI de l'export. Attention, même si ce fichier est optionnel, certaines propriétés ont des impacts directs sur les itinéraires proposés.
```

File: poi_fr.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+Ce fichier contient la liste des types de POI de l'export.
+
+Colonne | Type | Contrainte | Commentaire
+--- | --- | --- | ---
+`poi_type_id` | chaine | Requis | Identifiant unique et pérenne de la catégorie de POI
+`poi_type_name` | chaine | Requis | Nom de la catégorie de POI
+
+
+Le nombre de catégories et le contenu du champ `poi_type_id` est libre, mais
+des contraintes sont associées à l'utilisation dans Navitia et dans le NMP.
```
Comment (patochectp) [inline poi_fr.md]:
```suggestion
Le nombre de catégories et le contenu du champ `poi_type_id` est libre, mais
des contraintes sont associées à l'utilisation dans Navitia.
```

File: poi_fr.md [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+Propriété | Clé | Valeur (si fixée) | Commentaire
+--- | --- | --- | ---
+Rabattement en vélo | `bike_access_point` | 0 ou 1 | Indique si le POI (ou l'arrêt) est un point de rabattement en vélo vers le transport en commun. Si la propriété est absente, il est considéré que le POI n'est pas un point de rabattement. Cette propriété n'est pas utilisée par le moteur à ce jour. Elle est prévue en cas de besoin ulterieur.
+Rabattement en voiture | `car_access_point` | 0 ou 1 | Indique si le POI (ou l'arrêt) est un point de rabattement en voiture vers le transport en commun (ie. un parking ou un P+R). Si la propriété est absente, il est considéré que le POI n'est pas un point de rabattement.
+
+# Liste standard des propriétés
+
+La liste des propriétés supplémentaires sur un POI n'est pas fixe. Cependant,
+pour permettre un affichage standardisé sur les produits Canal TP, les
+propriétés utilisées sont normalisées.
```
Comment (patochectp) [inline poi_fr.md]:
```suggestion
La liste des propriétés supplémentaires sur un POI n'est pas fixe. Cependant,
pour permettre un affichage standardisé sur les produits Kisio Digital, les
propriétés utilisées sont normalisées.
```

File: poi_fr.md (hunk 3 above)
Comment (woshilapin) [reply poi_fr.md]: Done in 961ed4f180b20ef4c1c35f94d18ce6d45b98279a.

File: poi_fr.md (hunk 5 above)
Comment (woshilapin) [reply poi_fr.md]: Done in 961ed4f180b20ef4c1c35f94d18ce6d45b98279a.

File: poi_fr.md (hunk 4 above)
Comment (woshilapin) [reply poi_fr.md]: Done in 961ed4f180b20ef4c1c35f94d18ce6d45b98279a.

File: poi_fr.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+=======================
+
+# Introduction
+
+Les POI permettent d'enrichir Navitia avec des points de départ ou d'arrivée
+connus des usagers pour la rubrique "Itinéraires" ou la présentation des lieux
+publics ou des services à proximités d'un lieu.
+
+Cette spécification décrit le format d'échange de ces POI entre Fusio et
+Navitia2, et le format d'alimentation cible du nouveau NMM. Certains POI ont
```
Comment (patochectp) [inline poi_fr.md]: Fusio is no longer the only supplier of POI, and I’m not sure NMM still exists

Comment (woshilapin) [reply poi_fr.md]: I can just change it for something like this if you're fine with it?

```suggestion
Cette spécification décrit le format d'échange de ces POI vers
Navitia2. Certains POI ont
```

Comment (patochectp) [reply poi_fr.md]: That’s exactly what I was thinking.

File: poi_fr.md (hunk 1 above)
Comment (woshilapin) [reply poi_fr.md]: Outdated since this whole part has been removed.

File: poi_fr.md (hunk 2 above)
Comment (woshilapin) [reply poi_fr.md]: Outdated since this whole part has been removed.

File: poi_fr.md (hunk 6 above)
Comment (woshilapin) [reply poi_fr.md]: Done in 590207cda6692b3e3e93d8340354bb6021802cf1.

**Review by patochectp — APPROVED** [review]

**Review by datanel — COMMENTED** [review]
you should use a markdown linter like https://dlaa.me/markdownlint/
There is a lot of warnings/errors

**Comment by woshilapin** [comment]
> you should use a markdown linter like [dlaa.me/markdownlint](https://dlaa.me/markdownlint/)
> There is a lot of warnings/errors

Done in 8e82c90. Thanks.

**Review by datanel — APPROVED** [review]

**Review by prhod — APPROVED** [review]
Thanks! I noticed the changelog file is tagged `fr` whereas the content is in EN. Really minor and can be fixed later.

**Pull request merged**